keyboards/halokeys/elemental75/elemental75.c

```diff
@@ -0,0 +1,25 @@
+  /* Copyright 2022 Halokeys
+  *
+  * This program is free software: you can redistribute it and/or modify
+  * it under the terms of the GNU General Public License as published by
+  * the Free Software Foundation, either version 2 of the License, or
+  * (at your option) any later version.
+  *
+  * This program is distributed in the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  * GNU General Public License for more details.
+  *
+  * You should have received a copy of the GNU General Public License
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+#include "elemental75.h"
+#include "print.h"
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  /* If console is enabled, it will print the matrix position and status of each key pressed */
+  #ifdef CONSOLE_ENABLE
+      uprintf("KL: kc: 0x%04X, col: %u, row: %u, pressed: %b, time: %u, interrupt: %b, count: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed, record->event.time, record->tap.interrupted, record->tap.count);
+  #endif
+    return true;
+}
```

Comment:
This should be the kb level function, not the user level.  

And if it's the kb level, then it needs to to call the user level, as well.